src/clj/qbits/jet/server.clj

```diff
@@ -170,6 +170,7 @@ supplied options:
 
 
 * `:port` - the port to listen on (defaults to 80)
+* `:ports` - additional ports to listen on (defaults to [])
```

Comment:
We should have just the one entry `ports`.

Reply:
Done